digits/model/images/classification/test_views.py

```diff
@@ -825,6 +854,17 @@ class BaseTestCreatedTall(BaseTestCreated):
 class BaseTestCreatedCropInForm(BaseTestCreated):
     CROP_SIZE = 8
 
+class BaseTestCreatedDataAug(BaseTestCreated):
```

Comment:
I don't know... it feels like one class is enough (e.g. `Tall`) but you never know...